lib/mysql2.rb

```diff
@@ -13,16 +13,20 @@
     ENV['RUBY_MYSQL2_LIBMYSQL_DLL']
   elsif File.exist?(File.expand_path('../vendor/libmysql.dll', File.dirname(__FILE__)))
     # Use vendor/libmysql.dll if it exists, convert slashes for Win32 LoadLibrary
-    File.expand_path('../vendor/libmysql.dll', File.dirname(__FILE__)).tr('/', '\\')
+    File.expand_path('../vendor/libmysql.dll', File.dirname(__FILE__))
```

Comment:
I remember the backslashes being required, since the LoadLibrary call is native Windows and not Ruby.